source/regex.tex

```diff
@@ -1650,13 +1628,8 @@
 
 \begin{itemdescr}
 \pnum
-\requires
-\tcode{InputIterator} shall meet the
-\oldconcept{InputIterator} requirements\iref{input.iterators}.
```

Comment:
I see that this was dropped, whereas R1 does update it. #3463 keeps them and updates them, presumably because, unlike [algorithms], there's no blanket wording to this effect. It seems like [re] doesn't have such blanket wording, but I can see a lot of *`Iterator` uses and very few touches to those in this PR.

Reply:
It's brought in via the `string_type` construction in the "_Effects:_ Equivalent to".